HelpSource/Classes/PanAz.schelp

```diff
@@ -41,54 +38,88 @@ A control rate level input.
 
 argument::width
 
-The width of the panning envelope. Nominally this is 2.0 which
-pans between pairs of adjacent speakers. Width values greater
-than two will spread the pan over greater numbers of speakers.
-Width values less than one will leave silent gaps between
-speakers.
+The width of the panning envelope. Nominally this is code::2.0:: which pans between pairs of adjacent speakers.
+Values greater than code::2:: will spread the pan over greater numbers of speakers.
+Values less than code::1:: will leave silent gaps between speakers.
 
 
 argument::orientation
 
-Should be zero if the front is a vertex of the polygon. The first
-speaker will be directly in front. Should be 0.5 if the front
-bisects a side of the polygon. Then the first speaker will be the
-one left of center.
+Should be code::0:: if the front is a vertex of the spanning polygon.
+The first speaker will be directly in front. Should be code::0.5:: if the front
+bisects a side of the spanning polygon.
+Then the first speaker will be the one left of center.
 
 section::Comparison to Pan2
 
 Despite a certain similarity, link::Classes/Pan2:: and link::Classes/PanAz:: with 2 channels behave differently.
 
 code::
-// one full cycle for PanAz: from 0 to 1
-{ PanAz.ar(2, DC.ar(1), Line.ar(0, 1, 0.1)) }.plot(0.1)
-// one full cycle for Pan2: from -1 to 1 and back to -1
-{ Pan2.ar(DC.ar(1), EnvGen.kr(Env([-1, 1, -1], [0.05, 0.05]))) }.plot(0.1)
+// one full cycle for PanAz: from 0 to 2
+{ PanAz.ar(2, DC.ar(1), Line.ar(0, 2, 0.1), orientation: 0) }.plot(0.1)
+// comparing with Pan2
+{ Pan2.ar(DC.ar(1), Line.ar(0, 2, 0.1)) }.plot(0.1)
+// to create one full cycle for Pan2, move from -1 to 1 and back to -1
+{ Pan2.ar(DC.ar(1), EnvGen.ar(Env([-1, 1, -1], [0.05, 0.05]))) }.plot(0.1)
+::
+
+The same in one plot window:
+
+code::
+(
+{
+	[
```

Comment:
Probably we should make it more explicit that `Pan2` and `PanAz` are similar in that they expect a range `-1 ... 1` (useful if you use a UGen for panning), but `PanAz` (circular) makes a full round while `Pan2` (linear) moves from one pole to the other?

This would make it clearer:
```
(
{
	[
		// one full cycle for PanAz: from -1 to 1
		PanAz.ar(2, DC.ar(1), Line.ar(-1, 1, 0.1), orientation: 0),
		// compare with Pan2
		Pan2.ar(DC.ar(1), Line.ar(-1, 1, 0.1)),
		// to create one full cycle for Pan2, move from -1 to 1 and back to -1
		Pan2.ar(DC.ar(1), EnvGen.ar(Env([-1, 1, -1], [0.05, 0.05])))
	].flat;
}.plot(0.1, bounds:Rect(30, 30, 400, 800))
)
```